content/apps/continuous-deployment.md

```diff
@@ -0,0 +1,111 @@
+---
+menu:
+  main:
+    parent: apps
+title: Continuous Deployment
+weight: 10
+---
+
+Setting up continuous deployment allows you to automatically upload your
+changes to your desired enivronment.
+
+The first thing you need to do is setup a "deployer" user in your organization and give it permission to deploy to the desired space:
+
+```
+cf create-user ORGNAME_deployer RANDOM-PASSWORD
```

Comment:
We need to figure out what to do in this case. I dont want people using their personal accounts for this.